gui_qt/games_registry_actions.py

```
@@ -0,0 +1,209 @@
+    return RegistryActionResult(True, f"已更新 {md_path.name}。", rendered_games_md=True)
+
+
+def record_apply_batch_for_game_root(
+    workspace_root: Path,
+    *,
+    game_root: Path | str | None,
+    manifest_path: str | Path,
+) -> RegistryActionResult:
+    registry_path = _registry_file(workspace_root)
+    if not registry_path.is_file():
+        return RegistryActionResult(False, f"跳过 registry 记录：未找到 {registry_path.name}。")
```

Ruff (0.15.20)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)